database/utils/dataFormatter.js

```diff
@@ -0,0 +1,80 @@
+// formats Cassandra reviews to the client expected format
+var clientFormat = (reviews) => {
+  var formattedReviews = [];
+
+  // change between pictures folder (local) or s3 bucket
+  // var url = "https://adcobareviews.s3-us-west-1.amazonaws.com";
+  var url = 'pictures';
+
+  for (var i = 0; i < reviews.length; i++) {
+    formattedReview = {
+      _id: reviews[i].id,
+      memberInfo: {},
+      responderInfo: {},
+      reviewInfo: {}
+    };
+
+    var reviewpictures = imageFormatter(reviews[i].reviewpictures, url);
+
+    // transforms the review data to the expected format
+    formattedReview.memberInfo = {
+      memberContributions: reviews[i].membercontributions,
+      memberHelpful: reviews[i].memberhelpful,
+      memberId: reviews[i].memberid,
+      memberImg: `${url}${reviews[i].memberimg}`,
+      memberLocation: reviews[i].memberlocation,
+      memberUserName: reviews[i].memberusername
+    }
+    formattedReview.responderInfo = {
+      hotelId: reviews[i].hotelid,
+      responderClose: reviews[i].responderclose,
+      responderDate: reviews[i].responderdate,
+      responderName: reviews[i].respondername,
+      responderOrg: reviews[i].responderorg,
+      responderPicture: `${url}${reviews[i].responderpicture}`,
+      responderPosition: reviews[i].responderposition,
+      responderText: reviews[i].respondertext
+    }
+    formattedReview.reviewInfo = {
+      reviewDate: reviews[i].reviewdate,
+      reviewPictures: reviewpictures,
+      reviewRatings: ratingFormatter(reviews[i].reviewratings),
+      reviewText: reviews[i].reviewtext,
+      reviewTitle: reviews[i].reviewtitle,
+      reviewTripType: reviews[i].reviewtriptype
+    }
```

Comment:
I feel like there's probably a way to use destructuring to shrink this down a bit, but if it works then don't mess with it. It's perfectly readable.